usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -642,11 +642,41 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
             'program_activity': row.get('program_activity'),
         }
 
+        # individual FILE C D linkage
+        if not created and total_rows < 100000:
+            kwargs = {}
+            kwargs['recipient_id__isnull'] = False
+            if row.get('piid') is not None:
+                kwargs['piid'] = row.get('piid')
+            if row.get('parent_award_id') is not None:
+                kwargs['parent_award__piid'] = row.get('parent_award_id'),
+            if row.get('fain') is not None:
+                kwargs['fain'] = row.get('fain')
+            if row.get('uri') is not None:
+                kwargs['uri'] = row.get('uri')
+
+            award_queryset = awards.filter(**kwargs).values("id")[:2]
```

Comment:
So... is it ever possible for this to return >1 row?  In which case, the query isn't ordered, so the one assigned will be picked arbitrarily... is that OK?

Reply:
If the size of the queryset is not one, then it fails line 662 and we dont use the queryset.   Therefore the queryset doesnt have to be ordered.